CONTRIBUTING.rst

```diff
@@ -24,6 +24,20 @@ If you're not familiar with `Docker <https://docs.docker.com/>`_ and
 reading up on.
 
 
+Database
+========
+
+By default, you will have default data loaded for only the Pontoon Intro project.
+If you have a database dump, you can load it into your PostgreSQL database by running::
+
+  $ make loaddb DB_DUMP_FILE=path/to/my/dump
+
+Note that your database container needs to be running while you do that. You
+can start just the postgresql container by runnin::
+
+  $ docker-compose run postgresql
```

Comment:
Might be worth adding `-d` to that command so that the database container runs in the background? I know about it, but others might be frustrated by docker containers showing logs by default.